packages/core-transaction-pool/src/memory.ts

```diff
@@ -49,20 +58,31 @@ export class Memory {
     }
 
     public getExpired(maxTransactionAge: number): Interfaces.ITransaction[] {
+        const currentHeight: number = this.currentHeight();
+
+        if (!currentHeight) {
```

Comment:
This check can be removed, there's now always a block and therefore height always truthy.

Reply:
Y